src/components/RocketCard.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function RocketCard({
+  name, description, imageUrl, rocketID, reserved, onToggleReserve,
+}) {
+  const handleReserveRocket = () => {
+    onToggleReserve(rocketID);
+  };
+
+  return (
+    <div className="rocket-card">
+      <div className="image-container">
+        <img src={imageUrl} alt={name} className="rocket-image" />
+      </div>
+      <div className="rocket-details">
+        <h1>{name}</h1>
+        {reserved ? <p className="reserved-tag">Reserved</p> : null}
+        <p>{description}</p>
+        <button type="button" onClick={handleReserveRocket}>
+          {reserved ? 'Cancel Reservation' : 'Reserve Rocket'}
+        </button>
+      </div>
+    </div>
+  );
+}
```

Comment:
- [ ] [OPTIONAL] It would be better to use the colors right that will make things appear clean without struggle some people will struggles to see the the text if colors are not right